### DISCUSSION TITLE
do DNS Stamps require bootstraps?

### DISCUSSION THREAD

**Discussion by bcookatpcsd** [AdguardTeam/AdGuardHome#3487] [Q&A]
It looks as if everything in the stamp is encoded.

Does anything need to be resolved with an unecrypted recursor?

**Comment by ameshkov** [comment]
Regardless of whether you use a DNS stamp or a simple address, bootstrap will only be used if you have a domain name there. If you have an IP address, there's nothing to bootstrap.
